src/incubator/f5-bigip-ctlr/templates/f5-bigip-ctlr-clusterrole.yaml

```diff
@@ -1,10 +1,11 @@
+{{- if .Values.rbac.create -}}
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
-  name: f5-bigip-ctlr-clusterrole
+  name: {{ template "f5-bigip-ctlr.fullname" . }}
   labels:
     app: {{ template "f5-bigip-ctlr.name" . }}
-    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    chart: {{ template "f5-bigip-ctlr.chart" . }}
```

Comment:
Charts in helm are expected to be tied to a version. This naming convention is deliberate and should be retained. Operators may be expected to use multiple versions of a chart during rolling deployments, testing etc., and having the name explicitly reference the version should make tracking, debugging, and human error elimination easier.

@f5tsomanchi  The names of resources should be validated in a test environment

Reply:
@recursivelycurious - as said in the above comments,

The RBAC aspects look fine, however, this MR had not been merged yet due to difficulty around a testing environment to run the system tests. They may need to be modified to run the newer charts.

- Can you please elaborate the modifications needed to run newer charts ?

Reply:
https://helm.sh/docs/developing_charts/#charts-and-versioning

Reply:
If you look in the helpers `template "f5-bigip-ctlr.chart" .` is functionally equivalent to `.Chart.Name }}-{{ .Chart.Version | replace "+" "_"`